roundabout/functions/redirect.js

```diff
@@ -29,7 +39,6 @@ export async function redirectCarGet(request) {
   }
 
   const locateCar = carLocationResolver({ 
-    bucket: getEnv().BUCKET_NAME,
     s3Client: getS3Client(),
     expiresIn
   })
```

Comment:
we are currently not fallbacking on no location claims found, to attempt carpark-prod-0, probably we should?